git_machete/client.py

```diff
@@ -1512,7 +1512,10 @@ def __pick_remote(self, branch: str, is_called_from_traverse: bool) -> None:
         print("\n".join(f"[{index + 1}] {rem}" for index, rem in enumerate(rems)))
         msg = f"Select number 1..{len(rems)} to specify the destination remote " \
               "repository, or 'n' to skip this branch, or " \
-              "'q' to quit the traverse: "
+              "'q' to quit the traverse: " if is_called_from_traverse \
+              else f"Select number 1..{len(rems)} to specify the destination remote " \
```

Comment:
Hmmm okay we just don't present the `n` option to the user... but still, what happens if they provide sth else than `q`/`quit` or a number?

Reply:
Also, rel: https://github.com/VirtusLab/git-machete/issues/244